misc/config_tools/config_app/views.py

```diff
@@ -790,19 +790,19 @@ def upload_board_info():
                             os.remove(new_file)
                             continue
                         # update RDT->CLOS_MASK according to board xml
-                        if 'board' in xml_config_root.attrib and 'scenario' in xml_config_root.attrib \
-                                and 'uos_launcher' not in xml_config_root.attrib:
-                            cdp_enabled = xml_config.get_curr_value('hv', 'FEATURES', 'RDT', 'CDP_ENABLED')
-                            (num_clos_mask, num_mba_delay) = \
-                                get_num_of_rdt_res(filename.rsplit('.', 1)[0], cdp_enabled)
-                            elem_clos_max = xml_config.get_curr_elem('hv', 'FEATURES', 'RDT', 'CLOS_MASK')
-                            elem_mba_delay = xml_config.get_curr_elem('hv', 'FEATURES', 'RDT', 'MBA_DELAY')
-                            xml_config.delete_curr_elem('hv', 'FEATURES', 'RDT', 'CLOS_MASK')
-                            xml_config.delete_curr_elem('hv', 'FEATURES', 'RDT', 'MBA_DELAY')
-                            for i in range(num_clos_mask):
-                                xml_config.clone_curr_elem(elem_clos_max, 'hv', 'FEATURES', 'RDT')
-                            for i in range(num_mba_delay):
-                                xml_config.clone_curr_elem(elem_mba_delay, 'hv', 'FEATURES', 'RDT')
+                       # if 'board' in xml_config_root.attrib and 'scenario' in xml_config_root.attrib \
```

Comment:
Why not remove these altogether instead of commenting them out?

Reply:
Because there are some comment about MBA_DELAY, RD element. In order to quickly fix UI issue, firstly comment them out as a workaround.